packages/core/src/electron-browser/menu/electron-menu-contribution.ts

```diff
@@ -145,6 +153,14 @@ export class ElectronMenuContribution implements FrontendApplicationContribution
                 }
             }
         });
+        registry.registerCommand(ElectronCommands.CLOSE_WINDOW, {
+            execute: () => {
+                const focusedWindow = electron.remote.getCurrentWindow();
```

Comment:
```ts
const currentWindow = electron.remote.getCurrentWindow();

// and why not:
currentWindow.close(); // ?
```

Reply:
Thanks, I simplified the code and also the previous commands.

Each command was unnecessarily calling `electron.remote.getCurrentWindow()` and checking if it was `undefined`. Since `electron.remote.getCurrentWindow()` returns an `electron.BrowserWindow` it was unecessary to perform the check.